qa/tox.ini

```
@@ -18,3 +18,8 @@ commands=flake8 --select=F,E9 --exclude=venv,.tox
 basepython = python3
 deps = mypy
 commands = mypy {posargs:.}
+
+[testenv:import-tasks]
+basepython = python3
+deps = {env:TEUTHOLOGY_GIT:git+https://github.com/ceph/teuthology.git@master#egg=teuthology[coverage,orchestra,test]}
```

```suggestion
deps = {env:TEUTHOLOGY_GIT:git+https://github.com/ceph/teuthology.git@master}#egg=teuthology[coverage,orchestra,test]
```

so that we don't need to specify the packagename and extras_require in the env variables as well.